src/Components/Components.sln

```diff
@@ -15,10 +15,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Compon
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Components.WebAssembly.Tests", "WebAssembly\WebAssembly\test\Microsoft.AspNetCore.Components.WebAssembly.Tests.csproj", "{958AD6D2-174B-4B5B-BEFC-FA64B5159334}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Components.WebAssembly.Build", "WebAssembly\Build\src\Microsoft.AspNetCore.Components.WebAssembly.Build.csproj", "{E8AD67A4-77D3-4B85-AE19-4711388B62B1}"
```

Comment:
Will our build tasks be relocated to the `Microsoft.NET.Sdk.Razor` project now?

Reply:
Yes, that's my goal. We were also talking about having a Blazor WASM specific SDK, but that's still to be figured out.

Reply:
Got it. I do like the idea of having a Blazor WASM specific SDK. For a beginner, it helps make the distinction between Razor/Blazor Server/Blazor WASM a little bit easier to understand.